src/components/tooltips-hoc/DropPod.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as _ from 'underscore';
+import {positionValidator} from './positionValidation';
+
+export const DropPodPosition = {
+    bottom: 'bottom',
+    top: 'top',
+    left: 'left',
+    right: 'right',
+};
+
+export interface IDropPodProps {
+    isOpen: boolean;

```

Comment:
This should be optional since you have a default

```suggestion
    isOpen?: boolean;
```

Reply:
Its a state, we can let him always because its not specified on the component attributes